<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string notr="true">Ip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edit_ip"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_serach">
        <property name="text">
         <string notr="true">Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com_lang">
        <item>
         <property name="text">
          <string notr="true">English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">中文</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Deutsch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Español</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Português-Brasil</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Français</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">日本語</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">Русский</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_Country">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Country</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="country">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_RegionName">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Region Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="regionName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_City">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">City</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="city">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_Isp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Internet Service Provider</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="isp">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_Ip">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Ip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ip">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_tip">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string notr="true">A small tool made using the API provided by ip-api.com that requires internet connection. Please enter the IP address you want to query and click search</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_git">
      <property name="text">
       <string>github.com/dependon/iptools</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
